src/OpenApiType.php

```diff
@@ -64,18 +64,6 @@ public function __construct(
 
 	public function toArray(bool $isParameter = false): array|stdClass {
 		if ($isParameter) {
-			if ($this->type === 'boolean') {
-				return (new OpenApiType(
-					context: $this->context,
-					type: 'integer',
-					nullable: $this->nullable,
-					hasDefaultValue: $this->hasDefaultValue,
-					defaultValue: $this->hasDefaultValue ? ($this->defaultValue === true ? 1 : 0) : (null),
-					description: $this->description,
-					enum: [0, 1],
-				))->toArray($isParameter);
```

Comment:
@nickvergessen here `deprecated` was not passed on, that's why it appears now that the code is gone.